AWS/DirectoryInsights/get-jcdirectoryinsights.py

```diff
@@ -0,0 +1,76 @@
+import requests, datetime, json, boto3, os, gzip
+
+def jc_directoryinsights(event, context):
+    jcapikey = os.environ['JCAPIKEY']
+    incrementType = os.environ['incrementType']
+    incrementAmount = int(os.environ['incrementAmount'])
+    bucketName = os.environ['BucketName']
+
+    now = datetime.datetime.utcnow()
+
+    if incrementType == "minutes":
+        start_dt = now - datetime.timedelta(minutes=incrementAmount)
+    elif incrementType == "minute":
+        start_dt = now - datetime.timedelta(minutes=incrementAmount)
+    elif incrementType == "hours":
+        start_dt = now - datetime.timedelta(hours=incrementAmount)
+    elif incrementType == "hour":
+        start_dt = now - datetime.timedelta(minutes=incrementAmount)
+    elif incrementType == "days":
+        start_dt = now - datetime.timedelta(days=incrementAmount)
+    elif incrementType == "day":
+        start_dt = now - datetime.timedelta(days=incrementAmount)
+    else:
+        raise Exception("Unknown increment value.")
+        return
+
+    start_date = start_dt.isoformat("T") + "Z"
+    end_date = now.isoformat("T") + "Z"
+
+    fileStartDate = datetime.datetime.strftime(start_dt, "%m-%d-%YT%H-%M-%SZ")
```

Comment:
Can you change the time format to YYYY-MM-DDTHH:MM:SSZ  (RFC3339 Timestamp) so it's consistent with the timestamps in the events?